tensorflow/core/util/port.cc

```diff
@@ -116,4 +116,31 @@ bool IsMklEnabled() {
   return oneDNN_enabled;
 #endif  // ENABLE_MKL
 }
+
+bool IsZenDnnEnabled() {
+  static absl::once_flag once;
+#ifndef AMD_ZENDNN
+  return false;
+#else
+  static bool ZenDNN_enabled = false;
```

Comment:
Once we get all the integration PRs in and things work fine, we may consider setting the default value to `true` if the plug-in is present. As of now I think it's good to start with false. :)

Reply:
Yes, that is a good idea. Thank you for the quick approval.